src/libtrx/include/libtrx/game/anims/common.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "./types.h"
+
+#define BONE_SIZE 4
```

Comment:
This should be private to level.c reader IMO.

Reply:
Thanks, I've shifted it to `libtrx/game/level/common.h`. Currently it's still needed in `inject.c` and the two `level.c` files to set the correct `bone_idx` on objects. Once we get a common object reader in the libtrx level reader, we can shift the constant there too I think.